pkg/api/admin/default.go

```diff
@@ -0,0 +1,26 @@
+package admin
+
+import (
+	"github.com/Azure/go-autorest/autorest/to"
+)
+
+func setDefaults(oc *OpenShiftManagedCluster) {
```

Comment:
you could be even more explicit here setDefaultsForV7()

Reply:
It should not matter. Moved it to work with the internal model as I think doing this for external might cause issues. I think we dont care if it is done on v6 to v8+ or v7 to V9.